app/src/js/utils/table-config/collections.js

```diff
@@ -12,8 +12,11 @@ import BatchDeleteWithGranulesContent from '../../components/DeleteCollection/Ba
 export const tableColumns = [
   {
     Header: 'Name',
-    accessor: row => <Link to={`/collections/collection/${row.name}/${row.version}`}>{row.name}</Link>,
-    id: 'name',
+    accessor: 'name',
```

Comment:
It seems making `accessor` primitive can solve https://bugs.earthdata.nasa.gov/browse/CUMULUS-1842.  I'm testing it out and may make changes in additional places.  Thanks!

Reply:
Oh awesome, that actually makes a lot of sense!